src/rpc/rawtransaction.cpp

```diff
@@ -1262,6 +1263,552 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
     return result;
 }
 
+static std::string WriteHDKeypath(std::vector<uint32_t>& keypath)
+{
+    std::string keypath_str = "m/";
+    for (uint32_t num : keypath) {
+        bool hardened = false;
+        if (num & 0x80000000) {
+            hardened = true;
+            num &= ~0x80000000;
+        }
+
+        keypath_str += std::to_string(num);
+        if (hardened) {
+            keypath_str += "'";
+        }
+        keypath_str += "/";
+    }
+    return keypath_str.substr(0, keypath_str.size() - 1); // Drop last '/'
```

Comment:
Alternatively: move the `keypath_str += "/";` to the front of the loop and initialize with just `"m"`. That way you can avoid adding and then stripping the final slash.

Reply:
Done